packages/@react-spectrum/s2/stories/Tabs.stories.tsx

```diff
@@ -148,3 +148,55 @@ export const Dynamic: Story = {
     </div>
   )
 };
+
+function ExampleTest(props) {
```

Comment:
besides the terrible story name, do we want to have this example in our storybook docs or do we want to remove it?

Reply:
yeah, afaik, the issue only applies when the user customizes the layout. i can add a chromatic story using the play functions

Reply:
also a similar example will be on the docs page so it will be documented somewhere, it's nice to have something to test though but i'll add it to chromatic and remove it from our storybook docs